dev-java/libg/libg-7.0.0.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source test"
+MAVEN_ID="biz.aQute.bnd:aQute.libg:${PV}"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="A library to be statically linked. Contains many small utilities"
+HOMEPAGE="https://bnd.bndtools.org/"
+SRC_URI="https://github.com/bndtools/bnd/archive/${PV}.tar.gz -> aQute.bnd-${PV}.tar.gz"
+S="${WORKDIR}/bnd-${PV}"
+
+LICENSE="Apache-2.0 EPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+# aQute.bnd.test.jupiter does not exist
+# org.assertj.core.api.junit.jupiter does not exist
+RESTRICT="test" #839681
+
+CP_DEPEND="
+	dev-java/slf4j-api:0
+"
+
+DEPEND="${CP_DEPEND}
+	~dev-java/bnd-annotation-${PV}:0
+	dev-java/osgi-cmpn:8
+	>=virtual/jdk-17:*
+"
+
+RDEPEND="${CP_DEPEND}
+	>=virtual/jre-17:*
```

Comment:
is this really restricted to java 17+? if so, it would restrict all packages in the dependency chain to java 17+ even if they otherwise work with lower versions.

Reply:
Definitely
```
aQute.libg/src/aQute/libg/uri/URIUtil.java:161:  error: switch expressions are not supported in -source 11
```
Adding a comment to that ebuild.